### PR TITLE
Use flower for celery monitoring

### PR DESCRIPTION
**Pull request by Uxio0**
- Run flower on scheduler machine (default port 5555)

### CODE REVIEW

File: docker/web/celery/scheduler/run.sh [hunk 1]
```diff
@@ -10,8 +10,6 @@ else
 fi
 
 
-echo "==> $(date +%H:%M:%S) ==> Installing flower for monitoring <=="
-pip install 'celery[flower]'
-sleep 5
+sleep 10
```
Comment (fmrsabino) [inline docker/web/celery/scheduler/run.sh:13]: Why do we need this?

File: docker/web/celery/worker/run.sh [hunk 2]
```diff
@@ -16,7 +16,7 @@ echo "==> $(date +%H:%M:%S) ==> Setting up service... "
 python manage.py setup_service
 
 MAX_MEMORY_PER_CHILD="${WORKER_MAX_MEMORY_PER_CHILD:-2097152}"
-MAX_TASKS_PER_CHILD="${MAX_TASKS_PER_CHILD:-500000}"
+MAX_TASKS_PER_CHILD="${MAX_TASKS_PER_CHILD:-1000000}"
```
Comment (fmrsabino) [inline docker/web/celery/worker/run.sh:19]: What is the reason for this change?

Comment (fmrsabino) [inline docker/web/celery/worker/run.sh:19]: What is the reason for this change>

File: docker/web/celery/scheduler/run.sh (hunk 1 above)
Comment (Uxio0) [reply docker/web/celery/scheduler/run.sh:13]: Waiting for migrations to be run (this was already here, was removed this monday when doing testing with flower)

File: docker/web/celery/worker/run.sh (hunk 2 above)
Comment (Uxio0) [reply docker/web/celery/worker/run.sh:19]: Prevent memory leaks when running a lot of tasks

Comment (Uxio0) [reply docker/web/celery/worker/run.sh:19]: 1M is a number that we should not reach soon, but my plan is to implement metrics with flower/grafana/prometheus to check that

**Review by fmrsabino — APPROVED** [review]
